services/debugger/uploader.go

```diff
@@ -128,7 +134,7 @@ func (uploader *uploaderImpl[E]) uploadEvents(eventBuffer []E) {
 			req.URL.RawQuery = q.Encode()
 		}
 		req.Header.Set("Content-Type", "application/json;charset=UTF-8")
-		req.SetBasicAuth(config.GetWorkspaceToken(), "")
+		req.SetBasicAuth(uploader.hostedServiceSecret, "")
```

Comment:
This will break Open source or deployments using workspace token. We need to check the deployment type and then choose the right token.

Reply:
Fixed